features/orders.feature

```diff
@@ -0,0 +1,35 @@
+Feature: The order service back-end
+    As an Order Manager
+    I need a RESTful catalog service
+    So that I can keey track of all my orders
+
+Background:
+    Given the following orders:
+        | customer_id | status | 
+        | 5           | CONFIRMED    |
+        | 9           | SHIPPED      |
+        | 3           | SHIPPED      |
+        | 2           | DELIVERED    |
+        | 2           | CANCELLED    |
+        | 5           | IN_PROGRESS  |
+
+Scenario: The server is running
+    When I visit the "Home Page"
+    Then I should see "Order Demo RESTful Service" in the title
+    And I should not see "404 Not Found"
+
+Scenario: Create an Order
+    When I visit the "Home Page"
+    And I set the "Customer ID" to "5"
+    And I select "Confirmed" in the "Status" dropdown
+    And I press the "Create" button
+    Then I should see the message "Success"
+    And I should see "Today's date" in the "Created On" field
+    And I should see "Today's date" in the "Updated On" field
```

Comment:
I have different thoughts on these two fields. Since the UI site is for the order manager, this would be part of the information they want to see. I made these two fields read-only in the UI so it is not modifiable, but still provides the information. Let me know that you think

Reply:
I see, you can leave these fields unchanged. But the title for the updated on field says Gender. Please correct that in HTML.

Reply:
Will do